frontend/packages/dev-console/src/components/topology/shapes/ResourceIcon.scss

```diff
@@ -5,6 +5,9 @@
     fill: #fff !important;
   }
 }
+.odc-resource-icon-service {
+  fill: #6ca100;
```

Comment:
Since we dont have the exact color in PF4 global-css-variables, I have added the hex code here
cc: @serenamarie125